downstream/modules/platform/con-building-an-execution-environment-in-a-disconnected-environment.adoc

```diff
@@ -1,26 +1,26 @@
 to downloading the following requirements from the internet:
 
-* Ansible Galaxy (galaxy.ansible.com) or Automation Hub ({Console}) for any collections added to the EE image.
+* Ansible Galaxy (galaxy.ansible.com) or Automation Hub ({Console}) for any collections added to the {ExecEnvShort} image.
 * PyPI (pypi.org) for any python packages required as collection dependencies.
-* The UBI repositories (cdn.redhat.com) for updating any UBI-based EE images.
+* The UBI repositories (cdn.redhat.com) for updating any UBI-based {ExecEnvShort} images.
 ** The RHEL repositories might also be needed to meet certain collection requirements.
 * registry.redhat.io for access to the ansible-builder-rhel8 container image.
 
-Building an EE image in a disconnected environment requires a subset of all of these mirrored, or otherwise made available on the disconnected network.
-See xref:importing-collections-into-private-automation-hub_disconnected-installation[Importing Collections into Private Automation Hub] for information on importing collections from Galaxy or Automation Hub into a private automation hub.
+Building an {ExecEnvShort} image in a disconnected environment requires a subset of all of these mirrored, or otherwise made available on the disconnected network.
+See xref:importing-collections-into-private-automation-hub_disconnected-installation[Importing Collections into Private Automation Hub] for information on importing collections from {Galaxy} or {HubName} into a {PrivateHubName}.
 
 Mirrored PyPI content once transferred into the high-side network can be made available using a web server or an artifact repository like Nexus.
 
-The UBI repositories can be mirrored on the low-side using a tool like `reposync`, imported to the disconnected environment, and made available from Satellite or a simple web server (since the content is freely redistributable).
+The UBI repositories can be mirrored on the low-side using a tool like `reposync`, imported to the disconnected environment, and made available from Satellite or a simple web server (because the content is freely redistributable).
 
-The `ansible-builder-rhel8` container image can be imported into a private automation hub in the same way a custom EE can be imported.
-See xref:approving-the-imported-collection_disconnected-installation[Transferring a Custom EE Images Across a Disconnected Boundary] for details substituting `localhost/custom-ee` for
-`registry.redhat.io/ansible-automation-platform-21/ansible-builder-rhel8`. This will make the ansible-builder-rhel8 image available in the private automation hub registry along with the default EE images.
+The `ansible-builder-rhel8` container image can be imported into a {PrivateHubName} in the same way a custom {ExecEnvShort} can be imported.
+See xref:approving-the-imported-collection_disconnected-installation[Transferring Custom EE Images Across a Disconnected Boundary] for details substituting `localhost/custom-ee` for
```

Comment:
This link text should be updated to remove EE. You should probably check the file that the xref is pointing to.

Reply:
The way it's formatted right now reflects the module ID in the xref and the section title in the link text--they are different. I was thinking this is something we can fix while restructuring the modules. Is that what your comment is referring to?

Reply:
Sorry that wasn't clear. 
The xref itself is ok but the text between the square brackets needs to be fixed:
See xref:approving-the-imported-collection_disconnected-installation[**Transferring Custom Execution Environment Images Across a Disconnected Boundary**]

That subheading in proc-approving-the-imported-collection.adoc should also be changed to change EE -> Execution Environment.

Reply:
I updated the subheading; I've been trying to update the link text but the change isn't pushing through. Will revisit this after tech review.